internal/users/tests/tests.go

```diff
@@ -27,8 +33,15 @@ var (
 func OverrideDefaultOptions(t *testing.T, groupPath string, gpasswdCmd []string) {
 	t.Helper()
 
+	defaultOptionsMu.Lock()
+	defer defaultOptionsMu.Unlock()
```

Comment:
That doesn’t prevent having the default options changed under multiple tests, so there is still a datarace, no? You can have one tests changing the default options, another one doing it in parallel concurrently, and then, both functions being executed with the last `defaultOptions`. This is why those tests can’t be run in parallel (see the caller of `OverrideDefaultOptions` in pam_test.go: concurrency is disabled for it. This is why there is no need for a mutex on that one.

Reply:
I have no idea how I got there, tbh. I think I was just tired :joy: